src/annotator/guest.js

```diff
@@ -710,4 +718,19 @@ export default class Guest extends Delegator {
       this.toolbar.highlightsVisible = shouldShowHighlights;
     }
   }
+
+  /**
+   * Set whether the document can be doodled on
+   * 
+   * @param {boolean} shouldBeDoodleable
+   */
+  setDoodleability(shouldBeDoodleable) {
+    if(this.doodleCanvas) {
+      this.doodleCanvas.doodleable = !this.doodleCanvas.doodleable;
+      console.log("just set doodleable to: ", this.doodleCanvas.doodleable);
+    }
+    else {
+      alert("DOODLE CANVAS IS NULL!!");
+    }
```

Comment:
Extra alert, maybe console.error() here? Or remove entirely and don't care, depending on when this happens.